content/030_eksctl/launcheks.md

```diff
@@ -32,51 +32,32 @@ If you do see the correct role, proceed to next step to create an EKS cluster.
 ### Create an EKS cluster
 
 {{% notice warning %}}
-`eksctl` version must be 0.24.0 or above to deploy EKS 1.17, [click here](/030_eksctl/prerequisites) to get the latest version.
+`eksctl` version must be 0.48.0 or above to deploy EKS 1.19, [click here](/030_eksctl/prerequisites) to get the latest version.
 {{% /notice %}}
 
-Create an eksctl deployment file (eksworkshop.yaml) use in creating your cluster using the following syntax:
+Create your cluster using the following syntax:
 
 ```bash
-cat << EOF > eksworkshop.yaml
----
-apiVersion: eksctl.io/v1alpha5
-kind: ClusterConfig
-
-metadata:
-  name: eksworkshop-eksctl
-  region: ${AWS_REGION}
-  version: "1.17"
-
-availabilityZones: ["${AZS[0]}", "${AZS[1]}", "${AZS[2]}"]
-
-managedNodeGroups:
-- name: nodegroup
-  desiredCapacity: 3
-  instanceType: t3.small
-  ssh:
-    enableSsm: true
-
-# To enable all of the control plane logs, uncomment below:
-# cloudWatch:
-#  clusterLogging:
-#    enableTypes: ["*"]
-
-secretsEncryption:
-  keyARN: ${MASTER_ARN}
```

Comment:
It looks like secrets encryption would be disabled by this change. Is there a command line equivalent that would ensure this doesn't happen?

Reply:
Thank you @otterley for spotting issue. I have add support for secrets encryption using `eksctl` utility